flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -13,10 +13,50 @@
 from flexmeasures.data.schemas.sensors import SensorSchema
 from flexmeasures.data.models.generic_assets import GenericAsset
 from flexmeasures.tests.utils import QueryCounter
+from flexmeasures.utils.unit_utils import is_valid_unit
+
 
 sensor_schema = SensorSchema()
 
 
+@pytest.mark.parametrize(
+    "requesting_user, use_pagination",
+    [("test_supplier_user_4@seita.nl", False), ("test_supplier_user_4@seita.nl", True)],
+    indirect=["requesting_user"],
+)
+def test_fetch_sensors(
+    client,
+    setup_api_test_data,
+    requesting_user,
+    use_pagination,
+):
+    """
+    Retrieve all sensors.
+
+    Our user here is admin, so is allowed to see all sensors.
+    Pagination is tested only in passing, we should test filtering and page > 1
```

Comment:
I agree, please add another short test which filters. Maybe with this parameterization:

```
@pytest.mark.parametrize(
    "requesting_user, search-by,search-value",
    [
     ("test_supplier_user_4@seita.nl", "filter", "a-search-word")
     ("test_supplier_user_4@seita.nl", "unit", "a-unit")
    ]
```

Reply:
And how would `all_accessible` change for this user? Maybe it's a clear difference, which we can also easily test here.

Reply:
ill add these also

Reply:
From my test `all_accessible` doesn't make a notable difference